src/peft/peft_model.py

```diff
@@ -105,7 +105,21 @@
     PeftType.VBLORA: VBLoRAModel,
 }
 
-
+PEFT_TYPE_TO_PREFIX_MAPPING = {
```

Comment:
We have this exact same mapping in [`save_and_load`](https://github.com/huggingface/peft/blob/f4cf170a9c51d822f950cde0a0e1c87dc013403a/src/peft/utils/save_and_load.py#L360-L374). I would suggest that we factor this out into a constant in `utils/constants.py` and re-use it in both locations.